src/coreclr/tests/tests.targets

```diff
@@ -52,9 +52,7 @@
       <!-- Color output doesn't work well when capturing the output in the CI system -->
       <XunitArgs>$(XunitArgs) -nocolor</XunitArgs>
 
-      <CorerunExecutable Condition="'$(RunningOnUnix)' == 'true'"  >$(CORE_ROOT)\corerun</CorerunExecutable>
-      <CorerunExecutable Condition="'$(RunningOnUnix)' != 'true'">$(CORE_ROOT)\corerun.exe</CorerunExecutable>
-      <CorerunExecutable Condition="'$(TargetArchitecture)' == 'wasm'"> $(DotnetRoot)/dotnet</CorerunExecutable>
+      <CorerunExecutable>$(DotnetRoot)/dotnet</CorerunExecutable>
```

Comment:
Now that I'm reviewing this again - @jkotas do you know why we use corerun to run the xunit runner when a reasonably stable dotnet is also available? It feels like dotnet would be preferable to the corerun we just built in dotnet/runtime repo as well (otherwise we run the risk that something that a targeted test could have pinpointed nicely will instead show up as a weird crash in xunit).

Reply:
It is for historic reasons. I believe https://github.com/dotnet/runtime/issues/33068 tracks fixing that.

> otherwise we run the risk that something that a targeted test could have pinpointed nicely will instead show up as a weird crash in xunit

Yes, this would work if we used last stable .NET runtime to run the xunit. I agree it would be a nice place to be, but it is difficult to pull it off in the infrastructure - we would need to get two runtimes to the test machine - the stable one to run xunit and the current one to actually run the tests.

Reply:
Ah, I see. So this pretty much only works right now because we're not executing this on Helix. I guess that's fine until we'll need to scale this out and at that point we'll also have more resources to do this kind of work. Thanks!